kube/file-vault/file-vault-deployment.yml

```diff
@@ -0,0 +1,261 @@
+                  name: fmr-s3-stg
+                  {{ else }}
+                  name: fmr-s3-prod
+                  {{ end }}
+                  key: kms_key_id
+            - name: AWS_BUCKET
+              valueFrom:
+                secretKeyRef:
+                  {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+                  name: fmr-s3-bucket
+                  {{ else if eq .KUBE_NAMESPACE .UAT_ENV }}
+                  name: fmr-s3-uat
+                  {{ else if eq .KUBE_NAMESPACE .STG_ENV }}
+                  name: fmr-s3-stg
+                  {{ else }}
+                  name: fmr-s3-prod
+                  {{ end }}
+                  key: name
+            - name: AWS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: file-vault
+                  # encryption_key is randomly generated password used to encrypt the params that are returned by file-vault
+                  key: encryption_key
+            - name: AWS_EXPIRY_TIME
+              value: "120" # 2 minutes in seconds
+            - name: RETURN_ORIGINAL_SIGNED_URL
+              value: "no"
+            - name: ALLOW_GENERATE_LINK_ROUTE
+              value: "yes"
+            {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+            - name: DEBUG
+              value: "*"
+            {{ end }}
+          securityContext:
+            runAsNonRoot: true
+
+        - name: keycloak-proxy
+          image: quay.io/ukhomeofficedigital/gogatekeeper:2.9.2
+          resources:
+            limits:
+              memory: 1024Mi
+              cpu: 200m
+          envFrom:
+            - configMapRef:
+                {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+                name: {{ .APP_NAME }}-configmap-{{ .DRONE_SOURCE_BRANCH }}
+                {{ else }}
+                name: {{ .APP_NAME }}-configmap
+                {{ end }}
+          env:
+            - name: PROXY_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-client
+                  key: secret
+            - name: PROXY_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-client
+                  key: id
+            - name: PROXY_LISTEN
+              value: 127.0.0.1:3001
+            - name: PROXY_UPSTREAM_URL
+              value: "https://127.0.0.1:3000"
+            - name: PROXY_REDIRECTION_URL
+            {{ if eq .KUBE_NAMESPACE .PROD_ENV }}
+              value: https://fv.prod.{{ .APP_NAME }}.homeoffice.gov.uk
+            {{ else if eq .KUBE_NAMESPACE .STG_ENV }}
+              value: https://fv-{{ .APP_NAME }}.stg.sas.homeoffice.gov.uk
+            {{ else if eq .KUBE_NAMESPACE .UAT_ENV }}
+              value: https://fv-{{ .APP_NAME }}.uat.sas-notprod.homeoffice.gov.uk
+            {{ else if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+              value: https://fv-{{ .DRONE_SOURCE_BRANCH }}.branch.sas-notprod.homeoffice.gov.uk
```

Comment:
in `kube/file-vault/file-vault-ingress.yml` the host has been set as: 
`fv-{{ .DRONE_SOURCE_BRANCH }}.{{ .BRANCH_ENV }}.homeoffice.gov.uk`

which will produce different url as `BRANCH_ENV = sas-fmr-brach`

Reply:
updated.
We will use ACP recommendations,
fv-{{ .DRONE_SOURCE_BRANCH }}.branch.sas-notprod.homeoffice.gov.uk
The pattern is similar to the one used for branch ingress url used by FMR service
IF we use same pattern in future we will not see any missing policy admissions. 
for example I could request ACP to add policy admission. 
*.branch,sas-notprod.homeoffice.gov.uk